pkgs/applications/editors/vscode/extensions/vscode-utils.nix

```diff
@@ -100,10 +99,42 @@ let
     inherit lib buildEnv writeShellScriptBin extensionsFromVscodeMarketplace jq;
     vscodeDefault = vscode;
   };
+
+
+  toExtensionJsonEntry = ext: rec {
+    identifier = {
+      id = ext.vscodeExtUniqueId;
+      uuid = "";
+    };
+
+    version = ext.version;
+
+    location = {
+      "$mid" = 1;
+      fsPath = ext.outPath + "/share/vscode/extensions/${ext.vscodeExtUniqueId}";
+      path = location.fsPath;
+      scheme = "file";
+    };
+
+    metadata = {
+      id = identifier.uuid;
+      publisherId = "";
+      publisherDisplayName = ext.vscodeExtPublisher;
+      targetPlatform = "undefined";
+      isApplicationScoped = false;
+      updated = false;
+      isPreReleaseVersion = false;
+      installedTimestamp = 0;
+      preRelease = false;
+    };
```

Comment:
Do we actually need all these fields? For reference, this is what a typical entry generated by vscode looks like:

```json
  {
    "identifier": {
      "id": "james-yu.latex-workshop"
    },
    "version": "9.2.1",
    "location": {
      "$mid": 1,
      "fsPath": "/home/n/.vscode-oss/extensions/James-Yu.latex-workshop",
      "path": "/home/n/.vscode-oss/extensions/James-Yu.latex-workshop",
      "scheme": "file"
    }
  }
```

Reply:
I am on insiders so it may so that it generated a different schema.

Reply:
Good point. Could you test this PR on insiders?

Reply:
I've dropped that commit for now since it's not needed for this PR